src/react/projects/spark-react/src/SprkAccordion/components/SprkAccordionItem/SprkAccordionItem.js

```diff
@@ -61,23 +62,28 @@ class SprkAccordionItem extends Component {
 
     return (
       <li className={itemClassNames} data-id={idString} {...other}>
-        <a
+        <SprkLink
+          variant="unstyled"
           aria-controls={id}
-          className="sprk-c-Accordion__summary"
+          additionalClasses="sprk-c-Accordion__summary"
           data-analytics={analyticsString}
-          href="#nogo"
           onClick={this.toggle}
           aria-expanded={isOpen ? 'true' : 'false'}
+          href="#nogo"
```

Comment:
With the SprkLink component setting the default href prop to #. We can remove the href="#nogo" from this.